src/Core/Content/ImportExport/ImportExport.php

```diff
@@ -297,6 +297,7 @@ public function export(Context $context, ?Criteria $criteria = null, int $offset
             // default sorting
             $criteria->addSorting(new FieldSorting('createdAt', FieldSorting::ASCENDING));
         }
+        $criteria->addSorting(new FieldSorting('id'));
```

Comment:
Should this use something like the following instead of relying on a hardcoded `id` field (was based on the hardcoded `createdAt` field above)?
```suggestion
        foreach ($this->repository->getDefinition()->getPrimaryKeys() as $primaryKey) {
            $criteria->addSorting(new FieldSorting($primaryKey->getPropertyName()));
        }
```

Reply:
Hi, it's not necessary in this case. We can proceed with the PR. Thanks